app/models/vacancy.rb

```diff
@@ -98,13 +96,19 @@ def external?
   end
 
   def organisation
-    @organisation ||= organisations.one? ? organisations.first : organisations.first&.school_groups&.first
+    return organisations.first if organisations.one? || organisations.first&.school_group?
```

Comment:
Just some 💅🏻 

I think this would be more readable with a standard `if...else`

```
if organisations.one? || organisations.first&.school_group?
  organisations.first
else
  organisations.first&.school_groups&.first
end
  ```
  
Or you could create a predicate method for `organisations.one? || organisations.first&.school_group?` and use the one liner version.
Thoughts?

Reply:
This is a known pattern:
https://www.rubydoc.info/gems/rubocop/0.22.0/Rubocop/Cop/Style/GuardClause

Reply:
@cesidio Guard clauses are used to break early and avoid executing the rest of the method when it's unnecessary.

This wouldn't need a guard clause but a normal `if... else` as it would read better.

But happy to go with what you prefer as it's just aesthetics.